### PR TITLE
Add stacks blockchain support

### PR DESCRIPTION
**Pull request by emmanuelm41**
### 📝 Description

This PR intents to add stacks blockchain support for sending, receiving and checking balance features. 

### ❓ Context

- **Impacted projects**: `common, cli, desktop, mobile, cryptoassets` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### LLD
https://user-images.githubusercontent.com/10941036/199746197-df6e8e9c-9c7b-4250-9a80-0d885a47178b.mp4

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/cli/src/commands/send.ts [hunk 1]
```diff
@@ -81,7 +81,6 @@ export default {
                           deviceId: opts.device || "",
                         })
                         .pipe(
-                          map(toSignOperationEventRaw),
```
Comment (gre) [inline apps/cli/src/commands/send.ts:84]: not sure why this was removed but i think it's intended that we display to the console "row" version of these

Comment (gre) [inline apps/cli/src/commands/send.ts:84]: would it be ok to send a separate PR for this change? I'm not sure to understand it myself and would be good to have a dedicated PR of this "fix" (if it's a bug, please explain as it went under our radar 🙏 ) thanks

File: apps/ledger-live-desktop/src/renderer/families/stacks/MemoValueField.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  const onMemoValueChange = useCallback(
+    memoValue => {
+      onChange(bridge.updateTransaction(transaction, { memo: memoValue }));
+    },
+    [onChange, transaction, bridge],
+  );
+
+  // We use transaction as an error here.
+  // It will be usefull to block a memo wrong format
```
Comment (gre) [inline apps/ledger-live-desktop/src/renderer/families/stacks/MemoValueField.jsx:36]: shouldn't it be named as `.memo`? (as it maps to "memo" field, essentially the name of the subfields are opened to the implementation but it's good if we semantically maps it to fields)

File: apps/cli/src/commands/send.ts (hunk 1 above)
Comment (emmanuelm41) [reply apps/cli/src/commands/send.ts:84]: Hi gre, long time no interaction! It is good to work again on a PR haha.

Regarding what you said, that is intended to fix a bug with the send command. It was not working the way it was, as the signOperationEvent is expected not to be on raw version, but in normal version. I mentioned this to @haammar-ledger. Could you help me here Hakim?

Comment (emmanuelm41) [reply apps/cli/src/commands/send.ts:84]: It replaces the signOperation object to its raw format here 
https://github.com/Zondax/ledger-live/blob/134355d561bd8d576123d51f99cb5058be5721a4/libs/ledger-live-common/src/transaction/signOperation.ts#L60-L65

File: apps/ledger-live-desktop/src/renderer/families/stacks/MemoValueField.jsx (hunk 2 above)
Comment (emmanuelm41) [reply apps/ledger-live-desktop/src/renderer/families/stacks/MemoValueField.jsx:36]: Yes, it could be. However, if we need to define some specific types for memo in the future, it will be easier in this way. I could be renamed though.

File: apps/cli/src/commands/send.ts (hunk 1 above)
Comment (haammar-ledger) [reply apps/cli/src/commands/send.ts:84]: Hi @emmanuelm41, I remember that change yes, but I can't say I understood either why it was needed. :slightly_smiling_face: 

Testing locally, the cli `send` command currently works on my end.
As advised by @gre and since it is outside the scope of a coin integration, could you please consider opening a dedicated PR for this change, describing the issue it solves and how to reproduce it? Thanks.

**Comment by emmanuelm41** [comment]
Keeping PR alive....

**Comment by emmanuelm41** [comment]
Keeping PR alive....

**Comment by emmanuelm41** [comment]
@haammar-ledger should I create the PR again?
